templates/login_or_signup.tmpl

```diff
@@ -29,13 +29,13 @@
             <div class="mui-row">
                 <div class="mui-col-sm-6 mui-col-md-4 mui-col-sm-offset-3 mui-col-md-offset-4">
                     <ul class="mui-tabs__bar mui-tabs__bar--justified">
-                        <li class="mui--is-active">
+                        <li id="signup-tab">
```

Comment:
no need for this id anymore

Reply:
are you against having ids we don't use?

Reply:
yes

Reply:
to me it's no different from leaving unused variables all over. If i'm reading through this html and I see that id, I assume it's being used in the css or js.